features/json.yml

```
@@ -0,0 +1,12 @@
+name: JSON
+description: JavaScript Object Notation (JSON) is a syntax for serializing objects, arrays, numbers, strings, booleans, and null. The `JSON` JavaScript namespace provides static methods for parsing values from and converting values to JSON.
```

Suggesting to align the text with the other JS APIs (e.g., `Intl` starts with "The `Intl` API provides…)

```suggestion
description: The `JSON` API provides static methods for parsing values from and converting values to JavaScript Object Notation (JSON), a serialization format for objects, arrays, numbers, strings, Boolean values, and null.
```